src/hotspot/cpu/aarch64/sharedRuntime_aarch64.cpp

```diff
@@ -1571,6 +1551,8 @@ nmethod* SharedRuntime::generate_native_wrapper(MacroAssembler* masm,
     __ clinit_barrier(rscratch2, rscratch1, &L_skip_barrier);
     __ far_jump(RuntimeAddress(SharedRuntime::get_handle_wrong_method_stub()));
 
+    // Verified entry point must be aligned
```

Comment:
I think this comment should come around line 1539.

Reply:
Okay, I will fix it.